moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -703,9 +693,6 @@ bool ServoCalcs::applyJointUpdate(const Eigen::ArrayXd& delta_theta, sensor_msgs
     // Calculate joint velocity
     joint_state.velocity[i] =
```

Comment:
Can you please add some debug prints that I can use to prove the velocity calculation we have here is correct (i.e. print this calculation, the actual joint velocity, and the difference between the two)? This still looks wrong because the arm is not moving this entire distance commanded in the amount of time we are giving it. If it did then Servo would cause the arm to jerk from a start/stop state every update.

Reply:
please see the diagram in the PR description.

Reply:
I'll work on actually tracking the Cartesian position and using that feedback to suggest a latency parameter.

Reply:
I started a branch called "latency monitor" here. It's not ready to merge yet, though.

https://github.com/AndyZe/moveit2/tree/andyz/latency_monitor